Formula/trezor-agent.rb

```diff
@@ -5,12 +5,14 @@ class TrezorAgent < Formula
   homepage "https://github.com/romanz/trezor-agent"
   url "https://files.pythonhosted.org/packages/54/89/6de88988f9efa712cb7131cf3123a2a043c7956ee66e725b14a1ea4c27ec/trezor_agent-0.9.2.tar.gz"
```

Comment:
The latest version is v0.11.3 according to the github page , you should update that as well.

Reply:
0.9.2 is still the one on Pypi. Not sure what the situation is with that.

Reply:
Hmm, the release version on the github page and the ones on pythonhosted doesn't seem to match up with dates or anything.

Reply:
Seems possible they use GitHub as a sort of rolling development preview and PyPI for stable releases. Potentially worth asking upstream directly @menteb, unless you know the answer yourself.

Reply:
Possible. The 0.9.1 version and 0.9.2 version on pythonhosted lags by about a year though but 0.9.0 is release on the same day.

Reply:
I'll contact the developer